### PR TITLE
#1 - Added the boilerplate

### PR DESCRIPTION
**Pull request by kepop1**
What:
- Added the html, css, js boilerplate
- Added bootstrap, the CSS & JS
- Added jQuery
- Added jQuery UI
- Added Moment
- Updated the html title and added h1 title

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+  </head>
+  <body>
+    <h1>Project Time Tracker</h1>
```
Comment (surajverma2587) [inline index.html:17]: Why did you add this now?

Comment (kepop1) [reply index.html:17]: Meh, its worth it for now, put a div around it later

**Review by surajverma2587 — APPROVED** [review]
Looks good to me 👍
